src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/cas1/Cas1DummyController.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.controller.cas1
+
+import io.swagger.v3.oas.annotations.Operation
+import org.slf4j.LoggerFactory
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.AllocatedFilter
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.ApplicationSortField
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.AssessmentSortField
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.BookingSearchSortField
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1OutOfServiceBedSortField
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1SpaceBookingResidency
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas1SpaceBookingSummarySortField
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.PlacementRequestSortField
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.RiskTierLevel
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.SortDirection
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.SortOrder
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.TaskSortField
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.UserSortField
+
+@RestController
+@RequestMapping(path = ["/cas1"])
+class Cas1DummyController {
+  private val log = LoggerFactory.getLogger(this::class.java)
+
+  @Operation(
+    description = "This is a dummy endpoint that adds models missing from autogenerated swagger documentation. This is to keep the UI happy.",
```

Comment:
presumably this is temporary. Given this will appear in the swagger it may be worth mentioning this? e.g.

`This is a dummy endpoint that adds models missing from autogenerated swagger documentation that are expected by the UI. This will be removed once we remove controller auto generation`

Same for other dummy controller